tests/setup-debuggertests.cmd

```diff
@@ -0,0 +1,96 @@
+@if not defined __echo @echo off
+setlocal
+
+set __ThisScriptShort=%0
+set __ThisScriptFull=%~f0
+set __ThisScriptPath=%~dp0
+
+REM =========================================================================================
+REM ===
+REM === Parse arguments
+REM ===
+REM =========================================================================================
+
+set __OutputDir=
+set __Arch= 
+set __coreclr_bin_path=
```

Comment:
Make naming consistent (right now some are CapitalizedAndCamelCase, and others_have_underscores).